luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -196,17 +211,37 @@ class SeqrSVGenotypesSchema(SeqrGenotypesSchema):
 
     def _genotype_fields(self):
         is_called = hl.is_defined(self.mt.GT)
+        was_previously_called = hl.is_defined(self.mt.CONC_ST) & ~self.mt.CONC_ST.contains("EMPTY")
+        num_alt = self._num_alt(is_called)
+        prev_num_alt = hl.if_else(
+            was_previously_called,
+            PREVIOUS_GENOTYPE_N_ALT_ALLELES.get(hl.set(self.mt.CONC_ST)),
+            -1,
+        )
+        discordant_genotype = (num_alt != prev_num_alt) & (prev_num_alt > 0)
+        novel_genotype = (num_alt != prev_num_alt) & (prev_num_alt == 0)
         return {
             'sample_id': self.mt.s,
             'gq': self.mt.GQ,
             'cn': self.mt.RD_CN,
-            'num_alt': self._num_alt(is_called)
+            'num_alt': num_alt,
+            'prev_num_alt': prev_num_alt,
+            'new_genotype': hl.or_missing(is_called, discordant_genotype),
+            'new_call': hl.or_missing(is_called, ~was_previously_called | novel_genotype),
         }
 
     # NB: override this function here to mimic the existing null handling behavior.
     def _genotype_filter_samples(self, filter_):
         samples = super()._genotype_filter_samples(filter_)
-        return hl.or_missing(hl.len(samples) > 0, samples)            
+        return hl.or_missing(hl.len(samples) > 0, samples)
+
+    @row_annotation(fn_require=SeqrGenotypesSchema.genotypes)
+    def samples_new_call(self):
+        return self._genotype_filter_samples(lambda g: g.new_call)
+
+    @row_annotation(fn_require=SeqrGenotypesSchema.genotypes)
+    def samples_new_genotype(self):
```

Comment:
So looking into the desired behavior from the analysts, they want if they use the existing filter in seqr for "new calls" it to return variants that are either `new_call` or `new_genotype`. So we actually just want the `samples_new_call` field to include anything with either of those, instead of adding a new field